src/store/settings/settings-state.ts

```diff
@@ -50,7 +51,7 @@ export const settingsInitialState: SettingsState = {
   isManualBackupMade: true,
   isCloudBackupMade: true,
   applicationOpenCounter: 0,
-  isOnRampPossibility: false,
+  onRampOverlayState: OnRampOverlayState.Closed,
```

Comment:
what about state migration for this?

Reply:
I think there is no need in it because this value is added automatically if necessary

Reply:
But where should I make it? This property is used in `development` only here: https://github.com/madfish-solutions/templewallet-mobile/blob/8845aaa1d16cf904ea786267390ede594c005660/src/store/settings/settings-selectors.ts#L60 . The changes of this PR contain `isOnRampPossibility` variable but it's just an interpretation of `onRampOverlayState`